vars/createGithubIssue.groovy

```diff
@@ -1,6 +1,14 @@
 def call(Map args = [:]){
     def failureMessages = args.message
     List<String> failedComponents = []
+    def manifestPath = "${WORKSPACE}/manifests"
+
+    if (fileExists(manifestPath)) {
+        println("Manifest path exists, not checking out git repo")
+    } else {
+        println("Manifest path does not exists, checking out git repo")
+        git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
```

Comment:
We have been cleaning up these hardcoded `main` branches for a while. I would suggest let @zelinh code come in 1st before making changes. Since his change should reset my original stage thus postClean can be removed by then.

Reply:
@peterzhuamazon Can you post the PR or issue here? If that is long term we can get this change in as it is not affecting the build in anyway and would always point to main no matter what!
Thanks!

Reply:
Added already.

Reply:
So we don't need this PR anymore? If so can we enable GH issue creation to true again?
cc: @prudhvigodithi

Reply:
nope.

Reply:
OK I just realize @zelinh forget to remove the `postCleanUp()`......

Reply:
This PR should be changed to remove the postCleanUp() instead of adding a new git.

Reply:
@zelinh Can you take care of this since @rishabh6788 is out?

Reply:
Talked to @rishabh6788 and he will replace the scripted pipeline setup to stage setup with docker images, so build repo will be automatically pulled.